internal/render/sc.go

```diff
@@ -19,14 +22,15 @@ func (StorageClass) Header(ns string) Header {
 	return Header{
 		HeaderColumn{Name: "NAME"},
 		HeaderColumn{Name: "PROVISIONER"},
-		HeaderColumn{Name: "LABELS", Wide: true},
```

Comment:
labels and valid columns should remain in the header as they are standard cols across all resources.